.gitignore

```
@@ -14,7 +14,7 @@ __pycache__
 .mypy_cache
 
 # Local files
-emojis.zip
+guilds
```

Indicate that it should be a directory and not a file by adding `/` suffix.
```suggestion
guilds/
```